tests/kernels/test_encoder_decoder_attn.py

```diff
@@ -807,7 +808,7 @@ def test_encoder_only(
         assert_actual_matches_ideal(enc_test_params, enc_pckd_act_out)
 
 
-@pytest.mark.skipif(is_hip(), reason=STR_NOT_IMPL_ENC_DEC_ROCM_HIP)
+# @pytest.mark.skipif(is_hip(), reason=STR_NOT_IMPL_ENC_DEC_ROCM_HIP)
```

Comment:
why is this commented out? either keep it or remove it?

Reply:
removed, thanks